examples/dataset_configs.py

```diff
@@ -0,0 +1,137 @@
+from torchvision import transforms
+
+# the various types of dataset found in torchvision
+VISION_DATA_TYPES = {
+    'classification': list(('MNIST FashionMNIST KMNIST EMNIST QMNIST' +
+                            'FakeData LSUM ImageFolder DatasetFolder' +
+                            'ImageNet CIFAR10 CIFAR100 STL10 SVHN' +
+                            'USPS').split(' ')),
+    'captioning': list('CocoCaptions Flickr8k Flickr30k'.split(' ')),
+    #'detection': list('')
+    'segmentation': list('VOCSegmentation SBDataset'.split(' ')),
+    'action_reco': list('UCF101 HMDB51 Kinetics400'.split(' ')),
+}
```

Comment:
If we are manually including the names of particular items, then instead of doing `list(str.split())`, just write the strings out as elements of a pre-built list. 

(also, just as a tip, `str.split()` with zero arguments will automatically split along whitespace, no need to include the `str.split(' ')` character. Also, calling `.split()` creates a list as its return, no need to wrap it in `list()` funciton). 

```python
>>> 'hello world'.split()
['hello', 'world']

Reply:
Thanks for the tips !!